### ISSUE TITLE
numpy.test fails

### ISSUE THREAD

**Issue by louisabraham** [numpy/numpy#15786]
```pycon
>>> numpy.test(verbose=100)
NumPy version 1.18.1
NumPy relaxed strides checking option: True
======================================================== test session starts ========================================================
platform linux -- Python 3.8.2, pytest-5.3.5, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python
cachedir: .pytest_cache
rootdir: /home/louisabraham
collected 9328 items / 1 error / 180 deselected / 9147 selected                                                                     

============================================================== ERRORS ===============================================================
__________________________________________ ERROR collecting core/tests/test_regression.py ___________________________________________
ImportError while importing test module '/usr/lib/python3.8/site-packages/numpy/core/tests/test_regression.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.8/site-packages/_pytest/python.py:493: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
/usr/lib/python3.8/site-packages/py/_path/local.py:701: in pyimport
    __import__(modname)
/usr/lib/python3.8/site-packages/_pytest/assertion/rewrite.py:143: in exec_module
    exec(co, module.__dict__)
/usr/lib/python3.8/site-packages/numpy/core/tests/test_regression.py:20: in <module>
    from test.support import no_tracing
E   ModuleNotFoundError: No module named 'test'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
================================================= 180 deselected, 1 error in 18.42s =================================================
False
```

**Comment by charris** [comment]
Should be fixed in NumPy 1.18.2. Which is a good thing because it seems the presence of the standard Python module `test` cannot be relied upon.

**Comment by rossbar** [comment]
It is indeed fixed in 1.18.2
